src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -100,15 +100,15 @@ internal Project(Solution solution, ProjectState projectState)
         /// The language services from the host environment associated with this project's language.
         /// </summary>
         [Obsolete($"Use {nameof(Services)} instead.")]
-        public HostLanguageServices LanguageServices => _projectState.LanguageServices;
+#pragma warning disable CS0618 // Member is obsolete -- shouldn't be reported here https://github.com/dotnet/roslyn/issues/66409
```

Comment:
#pragmas shouldn't be required if this is the obsolete method calling the obsolete method.

Reply:
Yes, they shouldn't but they are see the bug.